### ISSUE TITLE
Accessibility Bug: Forced "peek" on command-click does not respect `editor.definitionLinkOpensInPeek` setting!

### ISSUE THREAD

**Issue by tgrushka** [microsoft/vscode#213020]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 
- OS Version: 

Steps to Reproduce:

Try to imagine being visually impaired / low vision when you are trying these steps. Try to imagine what it would be like when you expect a behavior to work a certain way, then suddenly your editor changes how it works, just because it thinks it knows how to be helpful to you. Now, when you're low vision, try to understand how this kind of abrupt change happening over and over again when using your editor, every time an update comes out and a new "feature" is introduced, and then trying to find documentation and figure out how to disable the "feature" can be extremely unnerving. Then imagine that if you do find a setting, it does not even work.

1. Open a Python project (re-definitions are very common in Python; could try with any other permissive language in which re-definitions are allowed);
2. Find a reference to a variable that has been defined and re-defined;
3. Command-click (control-click on Windows) the variable reference;
4. Note that peek comes up -- splits the editor window in two, instead of navigating to the definition;
5. Add `"editor.definitionLinkOpensInPeek": false` to settings;
6. Note the behavior has not changed!!

This was asked years ago, and still is not fixed, hence my lack of patience. https://stackoverflow.com/questions/41460845/disable-peek-in-visual-studio-code

I don't know what else to say. It is a definition link. I disabled peek on definition links. It does not respect my wishes and throws a split window, which is very annoying when you are low vision and want to see one window at a time. Meanwhile, what hack(s) are available to me to disable this behavior? My guess is none, without writing a complex extension. I just want my editor to behave, be stable, and stop throwing these curve balls at me constantly. Thank you for understanding.

**Comment by meganrogge** [comment]
Thanks for the feedback. 

I can see how the setting not being respected would be confusing. However, when I test the experience, with the peek view open, it does seem accessible. Do you agree with that or what would you change?

If it's simply the semantics that are confusing, would updating the setting description to more accurately reflect what happens for this case be a good solution in your opinion?

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!
